htdocs/commande/list.php

```diff
@@ -513,6 +513,9 @@
 		//$sql.= ' AND c.facture = 0'; // invoice not created
 		$sql .= ' AND ((c.fk_statut IN (1,2)) OR (c.fk_statut = 3 AND c.facture = 0))'; // validated, in process or closed but not billed
 	}
+	if ($search_status == -4) {	// To bill
```

Comment:
The property status "To bill" is no more a part of the status. It is a different non correlated property. There is already a filter. An order can have any status with to bill to 1 like it can has every status with to bill to 0

Reply:
yes it's a wrong comment. better is  " to validate and in progress"